.github/workflows/build_and_test.yml

```diff
@@ -277,6 +330,24 @@ jobs:
           python -m pip install -r requirements.txt
           python -m pytest --root-path=../../Dist/examples_bin
 
+      - name: Install Coverage Requirements
+        run: python3 -m pip install gcovr
+
+      - name: Process Coverage Files
+        shell: msys2 {0}
+        run: find . -name "*.gcno" -exec gcov -b -l -p -c {} +
+
+      - name: Create Coberture Report
+        run: gcovr -v -g -k -r . --gcov-ignore-parse-errors --exclude-throw-branches --filter Common --filter Pcap --filter Packet --xml -o coverage.xml
```

Comment:
For mingw run, I tried to use $GCOVR_FLAGS but generated empty report.

Reply:
Maybe because this job runs in powershell on Windows, you can try: `$env:GCOVR_FLAGS`

Reply:
Yes, it works (unfortunately partially). `gcovr` thinks they are directory 🙄. Check [this](https://github.com/seladb/PcapPlusPlus/actions/runs/4092523031/jobs/7057319238)

Reply:
Powershell is a weird language... try this:

```powershell
gcovr -v -g -k -r . $env:GCOVR_FLAGS.split() -o coverage.xml
```

Reply:
It works, thanks